src/utils/structuredData.ts

```diff
@@ -0,0 +1,75 @@
+import { AccordionProps } from 'blocks/AccordionBlock/AccordionBlock'
+
+export const structuredFAQPage = (
+  accordions: ReadonlyArray<AccordionProps>,
+) => ({
+  '@context': 'http://schema.org',
+  '@type': 'FAQPage',
+  mainEntity: accordions.map((item) => ({
+    '@type': 'Question',
+    name: item.title,
+    acceptedAnswer: {
+      '@type': 'Answer',
+      text: item.paragraph.html,
+    },
+  })),
+})
+
+export const structuredSoftwareApplication = () => [
+  {
+    '@context': 'http://schema.org',
+    '@type': 'SoftwareApplication',
+    name: 'Hedvig',
+    operatingSystem: 'IOS',
+    applicationCategory: 'FinanceApplication',
+    aggregateRating: {
+      '@type': 'AggregateRating',
+      ratingValue: '4.7',
+      ratingCount: '1200',
+    },
+  },
+  {
+    '@context': 'http://schema.org',
+    '@type': 'SoftwareApplication',
+    name: 'Hedvig',
+    operatingSystem: 'ANDROID',
+    applicationCategory: 'FinanceApplication',
+    aggregateRating: {
+      '@type': 'AggregateRating',
+      ratingValue: '4.4',
+      ratingCount: '180',
+    },
+  },
+]
+
+export const structuredWebSite = () => ({
+  '@context': 'http://schema.org',
+  '@type': 'WebSite',
+  name: 'Hedvig',
+  url: 'https://www.hedvig.com',
+  description:
+    'Hedvig är en ny typ av försäkring. Byggd på smart teknik, omtanke och sunt förnuft. Så att du kan få hjälp på sekunder, och ersättning på minuter.',
```

Comment:
Might there be a problem that this description is in Swedish but the `url` is the general for every market? Do we need to have one of these things for each locale?

Reply:
Agreed, I'm only replacing what we currently have in this PR so we can merge it quickly :-)